### DISCUSSION TITLE
Actualización

### DISCUSSION THREAD

**Discussion by JwSh-1** [skyline-emu/skyline#183] [Ideas]
Podrá agregar se una función para actualizar la aplicación de Skyline, dentro de el mismo emulador?

Tienen disponible algún sitio para ver la compatibilidad de los juegos con el emulador, O planean crearlo?

**Comment by bylaws** [comment]
Please speak in British English here. 

No hablo español 😨

**Reply by JwSh-1** [reply]
Can you add a function to update the Skyline application, within the same emulator?

Do you have a site available to see the compatibility of the games with the emulator, or do you plan to create it? 

((translator Google))

**Reply by bylaws** [reply]
Skyline doesn't play any games yet at all. It will be in the play store.

**Reply by Juancito1119** [reply]
Desearia que hubieran apks del juego
